internal/logger/zerolog.go

```diff
@@ -0,0 +1,81 @@
+package logger
+
+import (
+	"context"
+	"log/slog"
+
+	"github.com/rs/zerolog"
+)
+
+// ZerologHandler is a custom handler that adapts slog to zerolog
+type ZerologHandler struct {
+	logger zerolog.Logger
+}
+
+// newZerologHandler creates a new ZerologHandler
+func newZerologHandler(logger zerolog.Logger) *ZerologHandler {
+	return &ZerologHandler{logger: logger}
+}
+
+// Handle implements the slog.Handler interface
+func (h *ZerologHandler) Handle(ctx context.Context, record slog.Record) error {
+	event := h.logger.WithLevel(mapLevel(record.Level))
+
+	record.Attrs(func(attr slog.Attr) bool {
+		switch attr.Value.Kind() {
+		case slog.KindString:
+			event = event.Str(attr.Key, attr.Value.String())
+		case slog.KindInt64:
+			event = event.Int64(attr.Key, attr.Value.Int64())
+		case slog.KindFloat64:
+			event = event.Float64(attr.Key, attr.Value.Float64())
+		case slog.KindBool:
+			event = event.Bool(attr.Key, attr.Value.Bool())
+		case slog.KindAny:
```

Comment:
`default` and `slog.KindAny` is doing exact same thing which results in redundant code.

we can simplify it by removing `slog.KindAny` case and rely on `default`.